ClassCompass/Assignment.swift

```diff
@@ -10,12 +10,12 @@ import Foundation
 class Assignment {
     let id: Int
     let name: String
-    let dueDate: Date
+    let dueDate: Date?
```

Comment:
Why is dueDate optional?
I think we should add a new parameter that contains the dueOnDate. This would be the information that the user fills in for when they want to do the activity. That can be optional.

Reply:
so when checking the pull from the API, some dueDates are an address location, not a date. So when i was pulling the program would crash because it didn't identify the correct date type. So essentially it was a nil.